src/BarrelPlanarMPGDTracker_geo.cpp

```diff
@@ -159,7 +161,13 @@ static Ref_t create_BarrelPlanarMPGDTracker_geo(Detector& description, xml_h e,
       c_vol.setLimitSet(description, x_comp.limitsStr());
       c_vol.setVisAttributes(description, x_comp.visStr());
 
-      pv = m_vol.placeVolume(c_vol, Position(0, 0, thickness_sum + x_comp.thickness() / 2.0));
+      if (comp_name == "PCB") {
+        pcb_feb_ext = x_comp.offset();
+        pv          = m_vol.placeVolume(
+            c_vol, Position(0, -pcb_feb_ext / 2.0, thickness_sum + x_comp.thickness() / 2.0));
+      } else {
+        pv = m_vol.placeVolume(c_vol, Position(0, 0, thickness_sum + x_comp.thickness() / 2.0));
+      }
```

Comment:
Can we just mark every non-PCB layer to have
```
offset="-MPGDOuterBarrelModule_PCB_offset / 2"
```
and for PCB do
```
offset="MPGDOuterBarrelModule_PCB_offset / 2"
```
Then you don't need to have separate cases here and extra math for `z_placement`.